biz.aQute.bndlib/src/aQute/bnd/osgi/Clazz.java

```diff
@@ -159,6 +159,8 @@ public Map<String, Set<String>> getProfiles() throws IOException {
 		Java_22,
 		Java_23,
 		Java_24,
+		Java_25,
+		Java_26,
```

Comment:
You might want to go big here and add the next 4-6 versions. You can always go back later with properties files, if needed, once each version is released.

Adding 2 versions is only one year and people may use a Bnd version for a long time.